examples/routing_service/routing_service_with_security/dds_security_01/xml/PermissionsD2_DP.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
A short description of the permissions would be helpful.

Reply:
added comments to permissions file.